src/servicex_did_finder_lib/servicex_adaptor.py

```diff
@@ -88,6 +94,32 @@ def put_file_add(self, file_info):
             self.logger.error(f'After {attempts} tries, failed to send ServiceX App a put_file '
                               f'message: {str(file_info)} - Ignoring error.')
 
+    def put_file_add_bulk(self, file_list):
+        for chunk in chunks(file_list, 20):
```

Comment:
Should there really be a magic number of 20 here in this method? Maybe it should be an argument to `put_file_add_bulk`?

Reply:
hm. my comment from VScode disappeared... Anyhow, this number is a compromise between json size and number or requests generated. Given that some special datasets can have 20+ replicas, to be on the safe side I selected this number but everything from 20-100 will be OK. Making it an argument would only push ball along. We could do something fancier like summing up message size but that would be a bit of overkill.